epoptes-client/message

```diff
@@ -23,50 +23,58 @@
 # Public License can be found in `/usr/share/common-licenses/GPL".
 ############################################################################
 
-import gtk
+import gi
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk
 
-def MessageDialog(text, title="Epoptes", markup=True, icon="dialog-information"):
-    if icon == "dialog-error":
-        type = gtk.MESSAGE_ERROR
-    elif icon == "dialog-question":
-        type = gtk.MESSAGE_QUESTION
-    elif icon == "dialog-warning":
-        type = gtk.MESSAGE_WARNING
-    else:
-        type = gtk.MESSAGE_INFO
-    dlg = gtk.MessageDialog(None, 0, type, gtk.BUTTONS_CLOSE, text)
-    dlg.set_modal(True)
-    dlg.set_title(title)
-    dlg.set_property('use-markup', markup)
-    dlg.set_property('skip-taskbar-hint', False)
-    dlg.set_property('icon-name', icon)
-    dlg.run()
-    dlg.destroy()
 
+class MessageWindow(Gtk.Window):
 
-if __name__ == '__main__':
-    from sys import argv
+    def __init__(self, text, title="Epoptes", markup=True, icon_name="dialog-information"):
+        Gtk.Window.__init__(self, title=title, icon_name=icon_name)
 
-    if not (len(argv) >= 1 and len(argv) <= 5):
-        print "Usage: message text [title] [markup] [icon]"
-        exit(1)
+        spacing = 10
+        grid = Gtk.Grid(
+            column_spacing=spacing, row_spacing=spacing, margin_left=spacing, margin_top=spacing, margin_right=spacing,
+            margin_bottom=spacing)
```

Comment:
Have a look at [border_width](https://lazka.github.io/pgi-docs/Gtk-3.0/classes/Container.html#Gtk.Container.props.border_width), instead of setting all the margins to the same value

Reply:
Thanks, I updated it to use a single [margin](https://lazka.github.io/pgi-docs/Gtk-3.0/classes/Widget.html#Gtk.Widget.props.margin) instead.
(Isn't border normally used for [actual borders](https://www.w3schools.com/css/css_boxmodel.asp)?)